website/src/pages/showcase.js

```diff
@@ -99,11 +99,10 @@ const Showcase = () => {
         </a>
         <p>
           A curated list of{' '}
-          <a href="https://github.com/ReactNativeNews/React-Native-Apps">
+          <a href="https://github.com/ReactNativeNews/React-Native-Apps/#readme">
```

Comment:
```suggestion
          <a key="test" href="https://github.com/ReactNativeNews/React-Native-Apps">
```

Try this instead

Reply:
As Sebastien suggested, I have different `key` in mind, sorry for not being precise enough!

```suggestion
          <a key="app-list" href="https://github.com/ReactNativeNews/React-Native-Apps/#readme">
```
But please use something meaningful, so we do not need to push another commit and redeploy it, if the addition of `key` works

Reply:
> Try this instead

@slorber  thank you for the suggestions :),

Reply:
> > Try this instead
> 
> @slorber thank you for the suggestion :),

Reply:
> As Sebastien suggested, I have different `key` in mind, sorry for not being precise enough!
> 
> But please use something meaningful, so we do not need to push another commit and redeploy it, if the addition of `key` fixes the issue.
@Simek  no problem if i use `demo-apps` for the `key`? i pushed my last changes

Reply:
`demo-apps` is fine, thank you for the updates! 🙂

Reply:
thanks @Simek  !.